src/migrations/00012-add-device-types.sql

```diff
@@ -0,0 +1,302 @@
+  ),
+  (
+    'kitra520',
+    'RushUp Kitra 520 (DISCONTINUED)'
+  ),
+  (
+    'jetson-tx2',
+    'Nvidia Jetson TX2'
+  ),
+  (
+    'artik10',
+    'Samsung Artik 10 (DISCONTINUED)'
+  ),
+  (
+    'qemux86',
+    'QEMU X86 32bit (EXPERIMENTAL)'
+  ),
+  (
+    'parallella',
+    'Parallella (DISCONTINUED)'
+  ),
+  (
+    'beaglebone-green-wifi',
+    'BeagleBone Green Wireless (EXPERIMENTAL)'
+  ),
+  (
+    'intel-nuc',
+    'Intel NUC'
+  ),
+  (
+    'artik5',
+    'Samsung Artik 520 (DISCONTINUED)'
+  ),
+  (
+    'artik710',
+    'Samsung Artik 710 (DISCONTINUED)'
+  ),
+  (
+    'asus-tinker-board-s',
+    'Asus Tinker Board S (EXPERIMENTAL)'
+  ),
+  (
+    'beagleboard-xm',
+    'BeagleBoard-XM (EXPERIMENTAL)'
+  ),
+  (
+    'jetson-nano',
+    'Nvidia Jetson Nano (EXPERIMENTAL)'
+  ),
+  (
+    'beaglebone-pocket',
+    'PocketBeagle (EXPERIMENTAL)'
+  ),
+  (
+    'intel-edison',
+    'Intel Edison'
+  ),
+  (
+    'npe-x500-m3',
+    'NPE X500 M3'
+  ),
+  (
+    'nitrogen6x',
+    'Nitrogen 6X (DISCONTINUED)'
+  ),
+  (
+    'imx8m-var-dart',
+    'Variscite DART-MX8M (EXPERIMENTAL)'
+  ),
+  (
+    'beaglebone-green',
+    'BeagleBone Green (EXPERIMENTAL)'
+  ),
+  (
+    'odroid-c1',
+    'ODROID-C1+'
+  ),
+  (
+    'generic-aarch64',
+    'Generic AARCH64 (ARMv8) (EXPERIMENTAL)'
+  ),
+  (
+    'iot2000',
+    'Siemens IOT2000 (EXPERIMENTAL)'
+  ),
+  (
+    'orange-pi-one',
+    'Orange Pi One (EXPERIMENTAL)'
+  ),
+  (
+    'orbitty-tx2',
+    'CTI Orbitty TX2 (EXPERIMENTAL)'
+  ),
+  (
+    'kitra710',
+    'RushUp Kitra 710 (DISCONTINUED)'
+  ),
+  (
+    'raspberrypi3-64',
+    'Raspberry Pi 3 (using 64bit OS) (EXPERIMENTAL)'
+  ),
+  (
+    'beaglebone-black',
+    'BeagleBone Black'
+  ),
+  (
+    'bananapi-m1-plus',
+    'BananaPi-M1+ (EXPERIMENTAL)'
+  ),
+  (
+    'artik530',
+    'Samsung Artik 530 (DISCONTINUED)'
+  ),
+  (
+    'jetson-tx2-skycatch',
+    'Nvidia Jetson TX2 Skycatch (EXPERIMENTAL)'
+  ),
+  (
+    'hummingboard',
+    'Hummingboard'
+  ),
+  (
+    'cl-som-imx8',
+    'Compulab MX8M (EXPERIMENTAL)'
+  ),
+  (
+    'raspberry-pi2',
+    'Raspberry Pi 2'
+  ),
+  (
+    'cybertan-ze250',
+    'Cybertan ZE250 (DISCONTINUED)'
+  ),
+  (
+    'raspberrypi3',
+    'Raspberry Pi 3'
+  ),
+  (
+    'revpi-core-3',
+    'Revolution Pi Core 3'
+  ),
+  (
+    'n510-tx2',
+    'Aetina N510 TX2 (EXPERIMENTAL)'
+  ),
+  (
+    'skx2',
+    'SKX2 (EXPERIMENTAL)'
+  ),
+  (
+    'spacely-tx2',
+    'CTI Spacely TX2 (EXPERIMENTAL)'
+  ),
+  (
+    'srd3-tx2',
+    'Nvidia D3 TX2'
+  ),
+  (
+    'up-board',
+    'UP board'
+  ),
+  (
+    'ts7700',
+    'Technologic TS-7700 (DISCONTINUED)'
+  ),
+  (
+    'var-som-mx6',
+    'Variscite VAR-SOM-MX6 (EXPERIMENTAL)'
+  ),
+  (
+    'ts4900',
+    'Technologic TS-4900'
+  ),
+  (
+    'via-vab820-quad',
+    'VIA VAB 820-quad (DISCONTINUED)'
+  ),
+  (
+    'zynq-xz702',
+    'Zynq ZC702 (DISCONTINUED)'
+  )
+ON CONFLICT ON CONSTRAINT "device type_slug_key"
```

Comment:
NIT: How about updating the `name` `ON CONFLICT`?
If we don't, then the name will equal the slug, as inserted in the FULL_DT code above.
Not a blocker, since we will update them anyway with the next fetch from S3....